ydb/ydb_version.py

```diff
@@ -1 +1 @@
-VERSION = "2.10.0"
```

Comment:
The version not updates in v3 because bug.
It will fix while next publish v3.x version.